roles/iptables/templates/etc/iptables

```diff
@@ -29,6 +29,9 @@
 {% for ipaddr in groups.legacy_frontend -%}
 -A INPUT -p tcp -m tcp -m multiport --dports 8080,8888,8991 -m comment --comment "allowed TCP ports for Legacy" -j ACCEPT
 {% endfor -%}
+{% for ipaddr in [groups.legacy_frontend, groups.frontend, groups.archive, groups.pdf_gen] -%}
```

Comment:
Actually, I'm pretty sure all we need is `groups.nfs_connected` In fact, that is a super-set of the above groups, with in addition a couple I missed.

Reply:
Sorry I misled you initially :/